whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/FullTextIndex.java

```diff
@@ -70,7 +70,7 @@ public class FullTextIndex extends RebuildableIndex {
 
     static final String[] FIELD_NAMES;
 
-    private static final Set<AttributeType> SKIPPED_ATTRIBUTES = Sets.newEnumSet(Sets.newHashSet(AttributeType.CERTIF, AttributeType.CHANGED, AttributeType.SOURCE), AttributeType.class);
+    static final Set<AttributeType> SKIPPED_ATTRIBUTES = Sets.newEnumSet(Sets.newHashSet(AttributeType.CERTIF, AttributeType.CHANGED, AttributeType.SOURCE), AttributeType.class);
```

Comment:
Can you instead use the FILTERED_ATTRIBUTES and SKIPPED_ATTRIBUTES from ElasticIndexService as you already have that dependency in ElasticFulltextSearch ? This class is only for Lucene